website/source/docs/agent/options.html.md

```diff
@@ -1449,6 +1449,17 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
         single RPC call to a Consul server. See
         https://en.wikipedia.org/wiki/Token_bucket for more details about how
         token bucket rate limiters operate.
+    *   <a name="kv_max_value_size"></a><a href="#kv_max_value_size">
+        `kv_max_value_size`</a> - Configures the maximum number of
+        bytes for a kv request body to the [`/v1/kv`](/api/kv.html)
+        endpoint. This limit defaults to [raft's](https://github.com/hashicorp/raft)
+        suggested max size. **Note that increasing beyond this default can
```

Comment:
We _could_ put what the suggested max size is here, but it's nbd just a nitpick

Reply:
👍